### PR TITLE
Card

### PR DESCRIPTION
**Pull request by shraddha-amber**
## Card Documentation Page Preview
image

* Card with Badges
image

* Card with Dismiss Badge
image

* Card with Text Overlay
image

* Card with Horizontal Layout
image

* Card with Text Only
image

### CODE REVIEW

**Comment by saketkum11** [comment]
hey shraddha
there is some conflict in your code.

**Comment by shraddha-amber** [comment]
> hey shraddha there is some conflict in your code.



> hey shraddha there is some conflict in your code.

Thank you for pointing that out Saket, I have resolved the conflicts now

**Review by JowelTisso — APPROVED** [review]

File: Components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+@import "../../style.css";
```
Comment (anjali1102) [inline Components/Card/card.css:1]: can we /style.css
as it root file

**Comment by shobit1337** [comment]
Looks dope. Approved!

**Review by shobit1337 — APPROVED** [review]

**Review by anjali1102 — APPROVED** [review]
LGTM !

File: Components/Card/card.css (hunk 1 above)
Comment (shraddha-amber) [reply Components/Card/card.css:1]: i was not aware of that, thank you!

**Pull request merged**